Hunting Queries/MultipleDataSources/FirewallRuleChanges_using_netsh.yaml

```diff
@@ -0,0 +1,126 @@
+id: 3dc5dc8b-160b-407e-9925-24a91e3599df
+name: Rare firewall rule changes using netsh
+description: |
+  This query will show rare firewall rule changes using netsh utility by comparing rule names and program names from the previous day
+  with those from the historical chosen time frame.
+  - This technique was seen in relation to Solarigate attack but the results can indicate potential  malicious activity used in different attacks.
+  - The process name in each data source is commented out as an adversary could rename it. It is advisable to keep process name commented but 
+    if the results show unrelated false positives, users may want to uncomment it.
+  - Note also that the queries use the KQL "has_all" operator, which hasn't yet been documented officially, but will be soon.
+    In general, "has_all" will look for all string matches in the provided list of values. 
```

Comment:
Maybe a little more clear if we state it this way - In short, "has_all" will only match when the referenced field has all strings in the list.